site/write_ups/2022/MetaphorsBlogPost

```diff
@@ -0,0 +1,76 @@
+
+Example metaphors: “data is the new oil”, “data gold mine”, “data mining”, “knowledge extraction”, “data is the new electricity”
+
+This data ideology, borne of the business world, takes data as material – often as a natural resource like oil – which when first obtained is unsuitable for generating economic value. To turn the ‘raw’ economic potential into actual economic value, data must be processed (e.g., filtered, aggregated, visualised, and modelled), or as Peter Sondergaard, Senior Vice President of tech-hype-cycle consultancy Gartner put it in 2011,
+
+“Information is the oil of the 21st century, and analytics is the combustion agent.” (Gartner, 2011, cited in Haupt, 2016)
+
+The key claims in this data ideology are that data can be processed to obtain information that can be exchanged for financial gain, and that profit generation is necessarily good. It follows from these assertions that we should be recording/creating more data and processing it in order to drive economic growth. This is a clearly capitalist view of data, and its metaphors are typically used by data scientists or managers in sales pitches where they appeal for investment in a data-driven project by emphasising the economic value of data.
+
+By treating data as economic potential/resource, this ideology and its metaphors background the human side of data: data are recorded about humans and are used to make decisions that affect humans. This can have serious negative consequences for the data subjects as witnessed by numerous case studies (Marsan, 2012), such as the introduction of Google’s Street View, which was initially launched without the blurring of faces and vehicle numberplates, leading to people’s privacy being compromised (Shankland, 2008). 
+
+## Data as inherent and objective source of knowledge
+
+Example metaphors: “raw data”, “data collection”, “the data speaks for itself”, “the data doesn’t lie” 
+
+A large number of data metaphors evidence the view held in the wider data science community that data is, in some way, natural. Metaphors falling under the umbrella of data as a natural resource imagine data to be ‘out there’, a resource waiting to be tapped; the often-used phrase ‘data collection’ conjures the image that data are ‘lying on the ground’, ready to be picked up. Taking data as ‘natural’ so that they merely need to be ‘collected’ leads to the view that data are inherent to the world, and not that they need to be created. The resulting assertion of this data ideology is that data are an objective source of knowledge; the information they give about the world transcend individuals’ opinions and biases. This is accompanied by a similar moral claim as in the last data ideology: because data give us insight into how the world truly is, one should collect as much data as possible to best enable ‘data-driven’ decision making, regardless of the consequences of expanding the measurement of humans’ lives. 
```

Comment:
```suggestion
A large number of data metaphors evidence the view held in the wider data science community that data is, in some way, natural. 
Metaphors falling under the umbrella of data as a natural resource imagine data to be ‘out there’, a resource waiting to be tapped; the often-used phrase ‘data collection’ conjures the image that data are ‘lying on the ground’, ready to be picked up. 
Taking data as ‘natural’ so that they merely need to be ‘collected’ leads to the view that data are inherent to the world, and not that they need to be created. 
The resulting assertion of this data ideology is that data are an objective source of knowledge; the information they give about the world transcend individuals’ opinions and biases. 
This is accompanied by a similar moral claim as in the last data ideology: because data give us insight into how the world truly is, one should collect as much data as possible to best enable ‘data-driven’ decision making, regardless of the consequences of expanding the measurement of humans’ lives. 
```